web/src/IncrementButton.tsx

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react'
+
+type ComponentProps = {
+  incrementBy: number;
+}
+
+function IncrementButton(props: ComponentProps) {
+
+  const [count, setCount] = useState(0)
```

Comment:
Should each button have its own `count` in this case? It feels like being a button with a sole purpose of incrementing by a certain number should maybe update the app state, which would allow the button to just be a button that does something?

E.g you could pass a function as a prop to IncrementButton that mutates the state - and then within App.tsx pass in a function that updates the state? 

This is all in attempt to separate concerns 🙌